File: Elixir/Iniciante/1011.exs
raio = IO.gets("") |> String.trim() |> String.to_integer
pi = 3.14159
volume = (4.0/3.0) * pi * :math.pow(raio, 3)

IO.puts "VOLUME = #{:erlang.float_to_binary(volume, decimals: 3)}"

File: C/Iniciante/1011.c
#include <stdio.h>

int main() {

  double volume, pi = 3.14159;
  int raio;

  scanf("%d", &raio);
  
  volume = (4.0 / 3.0) * pi * pow(raio, 3);

  printf("VOLUME = %.3lf\n", volume);
  
  return 0;
  
}
